nilearn/plotting/html_surface.py

```diff
@@ -20,26 +20,79 @@ class SurfaceView(HTMLDocument):
     pass
 
 
+def _mix_colormaps(fg, bg):
+    """Mixes foreground and background arrays of RGBA colors.
+
+    Args:
+        fg (numpy.ndarray): Array of shape (n, 4), foreground colors
+        bg (numpy.ndarray): Array of shape (n, 4), background colors
+
+    Returns:
+        numpy.ndarray: Array of shape (n, 4), mixed colors
+    """
+    # Adapted from https://stackoverflow.com/questions/726549/algorithm-for-additive-color-mixing-for-rgb-values/727339#727339 # noqa: E501
+    mix = np.empty_like(fg)
+
+    mix[:, 3] = 1 - (1 - fg[:, 3]) * (1 - bg[:, 3])
+    mix[:, 0] = (
+        fg[:, 0] * fg[:, 3] / mix[:, 3]
+        + bg[:, 0] * bg[:, 3] * (1 - fg[:, 3]) / mix[:, 3]
+    )
+    mix[:, 1] = (
+        fg[:, 1] * fg[:, 3] / mix[:, 3]
+        + bg[:, 1] * bg[:, 3] * (1 - fg[:, 3]) / mix[:, 3]
+    )
+    mix[:, 2] = (
+        fg[:, 2] * fg[:, 3] / mix[:, 3]
+        + bg[:, 2] * bg[:, 3] * (1 - fg[:, 3]) / mix[:, 3]
+    )
+
+    return mix
```

Comment:
We should test that this does meaningful things.

Reply:
I added a few tests!